c-bindings-gen/src/types.rs

```diff
@@ -990,7 +990,6 @@ impl<'a, 'c: 'a> TypeResolver<'a, 'c> {
 		}
 		match full_path {
 			"[u8]" if is_ref => Some(("crate::c_types::u8slice::from_slice(", ")")),
-			"[usize]" if is_ref => Some(("crate::c_types::usizeslice::from_slice(", ")")),
```

Comment:
This is still referenced in `to_c_conversion_inline_from_path`. I'm not sure if its worth bothering to remove this? It still works, even if its currently unused in the exact upstream RL code.